test/runtests.jl

```diff
@@ -143,6 +143,10 @@ end
 @test UnitTests.A.B.C in Documenter.Utilities.submodules(UnitTests.A)
 @test UnitTests.A.B.C.D in Documenter.Utilities.submodules(UnitTests.A)
 
+## NavNode tests
+
+include("navnode.jl")
```

Comment:
Should we keep it in a separate file. If so, the location of the `include` might not be ideal.